docs/_docs/live-update/setup.md

```diff
@@ -25,7 +25,10 @@ To configure access to your documents, you need to generate an access token.
 
 2. Scroll down to **Personal access tokens** and click **Generate new token**
 
-3. Automotive Design for Compose requires a token with read-only File content access. All other scopes can be left set to "No access". Set the permissions appropriately, enter a name for the token and set the expiration, then click **Generate token**
+3. Automotive Design for Compose requires a token with read-only File content access and read-only Variables access if available. All other scopes can be left set to "No access". Set the permissions appropriately, enter a name for the token and set the expiration, then click **Generate token**
+    * Note: variable access is only available to Enterprise plan. If variable access is not 
+      available, developers can still get most of the functionalities work with DesignCompose 
```

Comment:
"most of the DesignCompose functionality"

Reply:
Done.